Collector/MigrationsCollector.php

```diff
@@ -82,6 +82,9 @@ public function getData()
         return $this->data;
     }
 
+    /**
+     * {@inheritDoc}
```

Comment:
That's not enough to silence the deprecation emitted by Symfony's error handler. You need to document the return type explicitly.

```suggestion
     * @return void
```